### PR TITLE
Update Understanding and configuring Cloudflare Page Rules (Page Rule…

### PR DESCRIPTION
**Pull request by ngayerie**
…s Tutorial).md

### CODE REVIEW

**Comment by ngayerie** [comment]
Hi @hannes-cf 
Would it be possible to review and approve if that looks ok for you?
Thanks!

File: content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Page Rules require a
 DNS record for your page rule to work. Page Rules won\'t apply to
 hostnames that don\'t exist in DNS or aren\'t being directed to
 Cloudflare.
+If needed, you can create a dummy A record pointing to `192.0.2.1` or a dummy AAAA record pointing to `100::` under [the DNS tab of your Cloudflare Dashboard](/dns/manage-dns-records/how-to/create-dns-records/).
```
Comment (hannes-cf) [inline content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md]: Same comment as here: https://github.com/cloudflare/cloudflare-docs/pull/9356/commits/6ea3c610802810852c05747e1b2a46bdeaef28e5#r1229802676

Comment (ngayerie) [reply content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md]: Updated!

File: content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md [hunk 2]
```diff
@@ -18,7 +18,22 @@ Page Rules require a
...
 Cloudflare.
-If needed, you can create a dummy A record pointing to `192.0.2.1` or a dummy AAAA record pointing to `100::` under [the DNS tab of your Cloudflare Dashboard](/dns/manage-dns-records/how-to/create-dns-records/).
+
+Depending on the record type, you can use different values as the target as a placeholder. Either one of these achieves the same outcome and you only need to create one:
+
+```
+www.example.com  A      192.0.2.1
+www.example.com  AAAA   2001:DB8::1
+www.example.com  CNAME  domain.example
+www.example.com  CNAME  domain.example
```
Comment (hannes-cf) [inline content/support/Page Rules/Understanding and configuring Cloudflare Page Rules (Page Rules Tutorial).md]: We have the CNAME record example twice, so need to remove one of them.

**Comment by ngayerie** [comment]
Hi @hannes-cf 
I've removed the extra CNAME

**Comment by ngayerie** [comment]
Hi @jedecf @kodster28 do you have any idea why the spell check is failling for this PR?

**Comment by pedrosousa** [comment]
There's some issue probably due to the filename (contains spaces and parentheses) which we'll have to address.
Don't worry, it's a non-blocking and people with write access will be able to merge once the PR is approved.

**Comment by ngayerie** [comment]
Hey @rprice74 , @zeinjaber 
Would it be possible for you to review and approve this PR?
Thanks!

**Review by rprice74 — APPROVED** [review]

**Pull request merged**